src/LandingPage.js

```diff
@@ -0,0 +1,308 @@
+import React, { Component } from "react";
+import "./App.css";
+import BurgerMenu from "./BurgerMenu";
+import hashtag from "./images/hashtag.svg";
+import AliceCarousel from "react-alice-carousel";
+import Swal from "sweetalert2";
+
+import image1 from "./images/img1.jpeg";
+import image3 from "./images/img3.jpeg";
+import image4 from "./images/img4.jpeg";
+import image4bis from "./images/img4bis.jpg";
+import image5 from "./images/img5.jpeg";
+import image7 from "./images/img7.jpeg";
+import image8 from "./images/img8.jpeg";
+import image9 from "./images/img9.jpeg";
+import image10 from "./images/img10.jpeg";
+import image12 from "./images/img12.jpeg";
+import image13 from "./images/img13.jpeg";
+import image14 from "./images/img14.jpeg";
+import image15 from "./images/img15.jpeg";
+import image16 from "./images/img16.jpeg";
+import image18 from "./images/img18.jpeg";
+import image21 from "./images/img21.jpeg";
+import image24 from "./images/img24.jpg";
+import image25 from "./images/img25.png";
+//import image27 from "./images/img27.png";
+import image26 from "./images/img26.png";
+import heart from "./images/heart.svg";
+import linkedinIcon from "./images/linkedin.svg";
+import mailIcon from "./images/envelope.svg";
+import githubIcon from "./images/github.svg";
+import "react-alice-carousel/lib/alice-carousel.css";
+//import swal from "sweetalert";
+
+class LandingPage extends Component {
+  opensweetalert() {
+    Swal.fire({
+      title: "Cant' wait to start to track?",
+      text: "#WORD is under construction and will be ready soon.",
+      input: "email",
+      inputLabel: "Enter your e-mail address to stay in the latest news",
+      inputPlaceholder: "Enter your email address",
+      confirmButtonText: "Notify me!",
+      validationMessage: "Thanks for subscribing! We'll get to you soon",
+      showDenyButton: "true",
+      denyButtonText: "No, thanks",
+      confirmButtonColor: "#b49d9a",
+      denyButtonColor: "black",
+      showCloseButton: true,
+      imageUrl:
+        "https://www.circle-movement.com/media/site/-558800900-1585586027/capture-decran-2020-03-30-a-18.32.37.png",
+      imageAlt: "Man exercising with gym rings",
+    }).then((email) => {
```

Comment:
> The modal: the user is able to enter his/her e-mail address. How can I retrieve the email address entered by the user in order to display it to confirm that everything went well?

Yes, right here in this `callback function`. If you log it, you'll see this is an object with a key `value`. So you can access it through `email.value`.

To store it somewhere, you could use a firebase that you already know how to use it. :)

Privacy note: When you publish this, it's important that you add a disclaimer somewhere saying that this is being stored on google firebase, for privacy reasons.

Reply:
Hi @sandrina-p , 
What does your privacy note mean exactly? What do I have to do?

Reply:
Leave a text somewhere saying that the email is saved on firebase. You could add that note on the modal bottom part for example or on the footer. For this project is an extra, not a big deal. But in real projects usually this type of info is communicated under a 'privacy policy' page